File: API/2SeeOrNot/src/main/java/ies/project/toSeeOrNot/dto/CinemaUser.java
package ies.project.toSeeOrNot.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;
import java.io.Serializable;

/**
 * @author Wei
 * @date 2021/1/2 18:28
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class CinemaUser implements Serializable {
    private String userName;

    @Email
    @NotBlank
    private String userEmail;

    @NotBlank
    private String password;

    private int role;

    private String avatar;

    private String location;

    private String description;

}

File: website/ticketApp/src/app/models/CinemaUser.ts
export class CinemaUser {
  userName: string;
  userEmail: string;
  password: string;
  role: number;
  avatar: string;
  location: string;
  description: string;


  constructor(userName: string, userEmail: string, password: string, location: string, description: string) {
    this.userName = userName;
    this.userEmail = userEmail;
    this.password = password;
    this.location = location;
    this.description = description;
  }
}
